playkit/src/main/java/com/kaltura/playkit/player/PlayerSettings.java

```diff
@@ -412,13 +412,21 @@ public PKRequestConfig getPKRequestConfig() {
 
     @Override
     public Player.Settings setMaxVideoSize(PKMaxVideoSize maxVideoSize) {
-        this.maxVideoSize = maxVideoSize;
+        ABRSettings abrSettings = getAbrSettings();
```

Comment:
I think that  - this.maxVideoSize &  this.maxVideoBitrate has to be removed from class and need to fix the getters accordingly

Reply:
Done. Removed the redundant variables and getters.